numpy/__init__.pyi

```diff
@@ -462,8 +471,15 @@ class float32(floating):
 
 class float64(floating):
     def __init__(self, __value: Optional[SupportsFloat] = ...) -> None: ...
+    # Renable the previously disabled `float` methods
+    # (as `np.float64` is actually a proper `float` subclass)
+    def __trunc__(self) -> int: ...  # type: ignore[override]
+    @classmethod
+    def fromhex(cls, __s: str) -> float: ...  # type: ignore[override]
```

Comment:
```suggestion
    def fromhex(cls, __s: str) -> float64: ...  # type: ignore[override]
```

Reply:
Also done in https://github.com/numpy/numpy/pull/17117/commits/445b6350f0b30054f5d74d114ffca1a0348126ad.